src/babashka/neil/version.clj

```diff
@@ -32,7 +132,15 @@ Bump the :version key in the project config.")))
 (defn print-version []
   (println "neil" meta/version))
 
-(defn neil-version [{:keys [opts]}]
-  (if (:help opts)
-    (print-help)
-    (run-root-command opts)))
+(defn tag-command? [args]
```

Comment:
This kind of handling can be done using the `cli/dispatch` table by first defining `{:cmds ["version" "tag"]}` and then `{:cmds ["version"]}` which will handle the rest. I prefer to do it over there instead of manually handling these args over here.

Reply:
This should be fixed now. I'm still using a single `neil-version/neil-version` function for each dispatch case so we don't have to repeat the `(if (:help opts) ...)` logic in each `run-*-command` function.